deeplearning4j-cli/deeplearning4j-cli-api/src/main/java/org/deeplearning4j/cli/subcommands/Train.java

```diff
@@ -59,294 +61,229 @@
-		} catch (IOException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
-		}
-
-		//this.debugLoadedConfProperties();
-
-		// get runtime - EXECUTION_RUNTIME_MODE_KEY
-
-		if (null != this.configProps.get( EXECUTION_RUNTIME_MODE_KEY )) {
-
-			this.runtime = (String) this.configProps.get(EXECUTION_RUNTIME_MODE_KEY);
-
-		} else {
-
-			this.runtime = EXECUTION_RUNTIME_MODE_DEFAULT;
-
-		}
-
-
-		// get output directory
-
-		if (null != this.configProps.get( OUTPUT_FILENAME_KEY )) {
-
-			this.outputDirectory = (String) this.configProps.get(OUTPUT_FILENAME_KEY);
-
-		} else {
-
-			// default
-			this.outputDirectory = "/tmp/dl4_model_default.txt";
-			//throw new Exception("no output location!");
-
-		}
-
-		// get input data
-
-		if ( null != this.configProps.get( INPUT_DATA_FILENAME_KEY )) {
-
-			//log.warn( "\nLOADED INPUT SRC\n\n" );
-			this.input = (String) this.configProps.get(INPUT_DATA_FILENAME_KEY);
-
-		} else {
-
-			// default
-			//this.input = "/tmp/dl4_model_default.txt";
-			throw new Exception("no input file to train on!");
-
-		}		
-			
-	/*
-		
-		if (null == this.configProps.get( OUTPUT_FILENAME_KEY )) {
-			
-			Date date = new Date() ;
-			SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss") ;
-			this.outputVectorFilename = "/tmp/canova_vectors_" + dateFormat.format(date) + ".txt";
-						
-		} else {
-			
-			// what if its only a directory?
-			
-			this.outputVectorFilename = (String) this.configProps.get( OUTPUT_FILENAME_KEY );
-			
-			if ( (new File( this.outputVectorFilename ).exists()) == false ) {
-				
-				// file path does not exist
-				
-				File yourFile = new File( this.outputVectorFilename );
-				if(!yourFile.exists()) {
-				    yourFile.createNewFile();
-				} 
-				
-			} else {
-				
-				if ( new File( this.outputVectorFilename ).isDirectory() ) {
-					
-					
-					Date date = new Date() ;
-					SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss") ;
-					//File file = new File(dateFormat.format(date) + ".tsv") ;
-					
-					this.outputVectorFilename += "/canova_vectors_" + dateFormat.format(date) + ".txt";
-					
-					
-				} else {
-					
-					// if a file that exists
-					
-					
-					(new File( this.outputVectorFilename )).delete();
-					
-					log.warn( "File path already exists, deleting the old file before proceeding..." );
-					
-					
-				}
-				
-				
-			}
-			*/
-		//log.warn( "Writing vectorized output to: " + this.outputVectorFilename + "\n\n" );
-
-		//}
-
-
-	}
-
-
-	public void debugLoadedConfProperties() {
-
-		Properties props = this.configProps; //System.getProperties();
-		Enumeration e = props.propertyNames();
-
-		log.warn("\n-- DL4J Configuration --");
-
-		while (e.hasMoreElements()) {
-			String key = (String) e.nextElement();
-			log.warn(key + " - " + props.getProperty(key));
-		}
-
-		log.warn("-- DL4J Configuration --\n");
-	}
-
+        if ( null != this.configProps.get( INPUT_DATA_FILENAME_KEY ))
```

Comment:
I think it would be better to replace references like this with guava equivalents String.isNullOrEmpty(<arg>). Will make a PR